build_tools/travis/install.sh

```diff
@@ -38,13 +38,20 @@ if [[ "$DISTRIB" == "conda" ]]; then
     # provided versions
     conda create -n testenv --yes python=$PYTHON_VERSION pip
     source activate testenv
-    conda install --yes nose pytest pytest-cov
+    conda install --yes nose pytest pytest-cov python-dateutil
```

Comment:
Checking a bit more we should change tzoffet by datetime.timedelta from the standard library